### ISSUE TITLE
worker_spec is set incorrectly

### ISSUE THREAD

**Issue by danpf** [dask/dask-jobqueue#280]
@guillaumeeb 

Hey I haven't updated dask-jobqueue in a while but after updating i've found a slight bug. that `worker_spec` (at least in `distributed` should be a dictionary of ids to workers(ie the len of the current # of workers), not a specification of workers to make (like `worker_spec = {'cores': 4, 'memory': '16 GB'}`)

currently if you have your minimum number of workers at 0, (and possibly anything less than 2, distributed/deploy/adaptive.Adaptive checks 
```
        try:
            current = len(self.cluster.worker_spec)
        except AttributeError:
            current = len(self.cluster.workers)
```
to get the number of current workers, and so we just forever sit and spin our wheels because Adaptive thinks we have enough workers already (in reality we haven't even scaled yet :o )

it looks like this happened on https://github.com/dask/dask-jobqueue/pull/184 ?

i can see what you're tying to do, maybe we just need to name jobquque 'worker_spec' to something else?

**Comment by guillaumeeb** [comment]
I believe this is a regression due to https://github.com/dask/distributed/pull/2675 and https://github.com/dask/distributed/pull/2735, the introduction of SpecificationCluster in Dask Distributed.

I wanted to open an issue for this, but I still didn't find the time to dive into SpecificationCluster changes. Obviously, your issue shows that dask-jobqueue is not compatible with this new interface.

So the fastest change is probably to rename this attribute. I encourage you to propose a PR for this. In the longer term, we need to be compatible with SpecificationCluster, I'll open an issue for that.

**Comment by danpf** [comment]
@guillaumeeb 
could you bump the version / make a release so that the conda version gets built?

**Comment by guillaumeeb** [comment]
Closed by #282.

Release done!
